staging/src/k8s.io/api/pkg/testing/deepcopy_test.go

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
how about putting it into the root package as roundtrip_test.go?

Reply:
Yeah, I wasn't sure where to put it, root package sgtm.